src/test/java/com/kodilla/ecommercee/ProductTestSuite.java

```diff
@@ -0,0 +1,121 @@
+package com.kodilla.ecommercee;
+
+import com.kodilla.ecommercee.dao.ProductDao;
+import com.kodilla.ecommercee.domain.Product;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import java.util.ArrayList;
+import java.util.Iterator;
+
+import static org.junit.Assert.assertEquals;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+public class ProductTestSuite {
```

Comment:
Metody z CrudRepository które są tutaj testowane są zaimplementowane w Springu więc tak naprawdę nie ma potrzeby ich testować. Jedyny sens to mieć prosty test typu save i findById żeby sprawdzić czy tabela istnieje w poprawnej postaci, czy mamy dobrze zdefiniowaną encje itp.